include/scheduler.h

```diff
@@ -1,23 +1,23 @@
 #pragma once
 
+#include <atomic>
```

Comment:
I assume this is needed now because it was transitive over "types.h" before?

Reply:
Actually this was only needed before the rebase onto epochs - removed.